src/dsp_tools/utils/connection_live.py

```diff
@@ -162,30 +236,34 @@ def post(
         # otherwise the client can get a timeout error while the API is still processing the request
         # in that case, the client's retry will have undesired side effects (e.g. duplicated resources),
         # and the response of the original API call will be lost
-        timeout = 60
+        timeout = timeout or 60
```

Comment:
wouldn't it make more sense to set a default value for the parameter in the method signature? And then, you could move the long inline comment into the docstring

Reply:
I would prefer to keep the default value hard-coded in as few places as possible (and the way python implements protocols, it's actually a bit awkward to have defaults in parameters...), but this ties in into some things I was going to suggest as future improvements anyways. I'd leave it for now.